docker-compose.prod.yml

```
@@ -0,0 +1,9 @@
+# docker-compose.prod.yml
+version: "3.8"
+services:
+  web:
+    build:
+      context: .
+      dockerfile: Dockerfile.prod
+    ports:
+      - "80:80"
```

```suggestion
      - "80:80"
    environment:
      - VITE_MYFIN_BASE_API_URL=https://api.xyzwebsite.com 
```

This should address the concern raised by @duckimann, right?

---

actually, no. the `VITE_MYFIN_BASE_API_URL` only available for you to set at build time. and the environment you set is run time. same as you build on a machine without setting the api url then copied it to another machine and set the env over there

---

[i would add a reverse proxy](https://github.com/afaneca/myfin/issues/412#issuecomment-2635636539) to nginx config but can't get it to work last time